chains/dodotestnet/addresses.yaml

```diff
@@ -0,0 +1,13 @@
+staticMerkleRootMultisigIsmFactory: "0xDea86e6d55173d1E767797a856102cadf8B0A0a9"
```

Comment:
These need to be sorted alphabetically. If you run yarn lint:fix it will fix it for you